### PR TITLE
 fix(header): add popover a11y attributes

### LINKED ISSUE #716: Shell popovers don't have accessible labels
## Bug - the Notifications and Profile popovers in the Shell component aren't labelled. 

The Notification and Profile popovers in the top right corner of the `Shell` component don't have a label. 

image

Here's the issue that accessibility testers came back to us with, which applies to both popovers: 

image

---

I believe that hardcoding an `aria-label` for both popovers would solve the issue. 

It would need to be added for the notification popover: 

https://github.com/carbon-design-system/ibm-security/blob/4606908cbb67e326fcd416e4fe1907f07f6d388c/src/components/Header/Header.js#L300

and the profile popover: 

https://github.com/carbon-design-system/ibm-security/blob/4606908cbb67e326fcd416e4fe1907f07f6d388c/src/components/Header/Header.js#L175

### Optional information

**Version -** `1.28.0`

### PR DESCRIPTION
**Pull request by SimonFinney**
## Affected issue

Resolves #716

## Proposed change

Add `aria-labelledby` attributes for `Header` popover

### CODE REVIEW

**Comment by SimonFinney** [comment]
Hi @benborgers - Let me know your thoughts on this one to see if it satisfies your requirements.

Also, if you're interested in participating as an official reviewer / contributor, let me know and I'll coordinate access for the org 🎉

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -221,7 +222,10 @@ export default class Header extends Component {
               large
             />
             <div>
-              <span className={`${namespace}__popover__profile__header__title`}>
+              <span
+                id="profile"
```
[the remark this answers is not in the document]
Comment (SimonFinney) [reply src/components/Header/Header.js:226]: Great point!

**Review by jendowns — APPROVED** [review]

**Pull request merged**